deploy/kube-templates/glusterblock-provisioner.yaml

```diff
@@ -0,0 +1,28 @@
+---
+kind: List
+apiVersion: v1
+items:
+- kind: Deployment
+  apiVersion: extensions/v1beta1
+  metadata:
+    name: glusterblock-provisioner
+    labels:
+      glusterfs: glusterblock-provisioner-deployment
+    annotations:
+      description: Defines how to deploy the glusterblock provisioner pod.
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        name: glusterblock-provisioner
+        labels:
+          glusterfs: glusterblock-provisioner-pod
+      spec:
+        serviceAccountName: glusterblock-provisioner
+        containers:
+        - name: glusterblock-provisioner
+          image: gluster/glusterblock-provisioner:latest
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: PROVISIONER_NAME
+            value: glusterblock-provisioner
```

Comment:
I just went with what you had in the templates you provided. And per our earlier discussions, I see no need to make this configurable if it is very unlikely to change. If it ever changes, we can just update the scripts as I imagine this will not be a common occurrence. Does it matter what the value is, or is `gluster.org/glusterblock` correct? If so, why is it not so in the templates you have? :)